cmd/controller/main.go

```diff
@@ -50,6 +50,63 @@ var (
 	workerCount   int
 )
 
+// TODO:desc
+type controllerContext struct {
```

Comment:
It's weird calling this context. Not really sure if that's a common thing within Kubernetes, but these days, when I see something called context I mostly expect it to use the `context` package. :thinking:

Reply:
sure we can change the name, I don't have strong naming preferences.